### PR TITLE
restored content + other fixes

### LINKED ISSUE #5665: Doc bug? Table seems cut off
The first table seems to be cut off, as it just says "A secure and..." and then nothing, in the description. Also noticed that the first table lacks the UdpBinding, so it looks like it may have been cut off in a previous update. Thanks for being awesome, docs.ms!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2387192f-264a-c1c0-ab59-ffa1d4f5bd53
* Version Independent ID: 79f2524b-bfb4-fd79-bbe7-46f7584c3a53
* Content: [System-Provided Bindings](https://docs.microsoft.com/en-us/dotnet/framework/wcf/system-provided-bindings#feedback)
* Content Source: [docs/framework/wcf/system-provided-bindings.md](https://github.com/dotnet/docs/blob/master/docs/framework/wcf/system-provided-bindings.md)
* Service: **unspecified**
* Product: **dotnet-framework**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #5665 

/cc @bradygaster 

Fixes done:
- Restored content missed during migration
- Acrolinx suggestions
- Fixed values for WebSocketTransportUsage

Better to [hide whitespace changes](https://github.com/dotnet/docs/pull/5773/files?utf8=%E2%9C%93&diff=split&w=1) on the diff to better see changes

### CODE REVIEW

File: docs/framework/wcf/system-provided-bindings.md [hunk 1]
```diff
@@ -1,86 +1,89 @@
...
-|<xref:System.ServiceModel.WSHttpContextBinding>|WS|Transport, (Message), Mixed|(None), Reliable Session, Security Session|(None), Yes|n/a|Text, (MTOM)|No|  
-|<xref:System.ServiceModel.UdpBinding>|.NET **Note:**  Interoperability can be achieved by implementing the standard SOAP-over-UDP spec which this binding implements.|(None)|(None)|(None)|n/a|(Text)|No|  
-  
+> Never use duplex contracts with bindings that do not support security or that have security disabled unless you secure the data by some other means.
+
+The following bindings ship with WCF:
+
+|Binding|Configuration Element|Description|
+|-------------|---------------------------|-----------------|
+|<xref:System.ServiceModel.BasicHttpBinding>|[\<basicHttpBinding>](../configure-apps/file-schema/wcf/basichttpbinding.md)|A binding that is suitable for communicating with WS-Basic Profile conformant Web services, for example, ASP.NET Web services (ASMX)-based services. This binding uses HTTP as the transport and text/XML as the default message encoding.|
```
Comment (rpetrusha) [inline docs/framework/wcf/system-provided-bindings.md]: nit: hyphen between "Profile" and "conformant"

File: docs/framework/wcf/system-provided-bindings.md [hunk 2]
```diff
@@ -1,86 +1,89 @@
...
+|<xref:System.ServiceModel.WSHttpBinding>|[\<wsHttpBinding>](../configure-apps/file-schema/wcf/wshttpbinding.md)|A secure and interoperable binding that is suitable for non-duplex service contracts.|
+|<xref:System.ServiceModel.WSDualHttpBinding>|[\<wsDualHttpBinding>](../configure-apps/file-schema/wcf/wsdualhttpbinding.md)|A secure and interoperable binding that is suitable for duplex service contracts or communication through SOAP intermediaries.|
+|<xref:System.ServiceModel.WSFederationHttpBinding>|[\<wsFederationHttpBinding>](../configure-apps/file-schema/wcf/wsfederationhttpbinding.md)|A secure and interoperable binding that supports the WS-Federation protocol that enables organizations that are in a federation to efficiently authenticate and authorize users.|
```
Comment (rpetrusha) [inline docs/framework/wcf/system-provided-bindings.md]: There is one too many "that" here. I think "WS-Federation protocol, which enables..."

File: docs/framework/wcf/system-provided-bindings.md [hunk 3]
```diff
@@ -1,86 +1,89 @@
...
+|<xref:System.ServiceModel.NetMsmqBinding>|[\<netMsmqBinding>](../configure-apps/file-schema/wcf/netmsmqbinding.md)|A queued binding that is suitable for cross-machine communication between WCF applications.|
+|<xref:System.ServiceModel.NetPeerTcpBinding>|[\<netPeerTcpBinding>](../configure-apps/file-schema/wcf/netpeertcpbinding.md)|A binding that enables secure, multiple machine communication.|
+|<xref:System.ServiceModel.MsmqIntegration.MsmqIntegrationBinding>|[\<msmqIntegrationBinding>](../configure-apps/file-schema/wcf/msmqintegrationbinding.md)|A binding that is suitable for cross-machine communication between a WCF application and existing Message Queuing applications.|
+|<xref:System.ServiceModel.BasicHttpContextBinding>|[\<basicHttpContextBinding>](../configure-apps/file-schema/wcf/basichttpcontextbinding.md)|A binding that is suitable for communicating with WS-Basic Profile conformant Web services that enables HTTP cookies to be used to exchange context.|
```
Comment (rpetrusha) [inline docs/framework/wcf/system-provided-bindings.md]: Two occurrences of "that" make this sentence hard to parse. Better: "A binding suitable for communicating...that enables..."

File: docs/framework/wcf/system-provided-bindings.md [hunk 4]
```diff
@@ -1,86 +1,89 @@
...
+|<xref:System.ServiceModel.NetTcpContextBinding>|[\<netTcpContextBinding>](../configure-apps/file-schema/wcf/nettcpcontextbinding.md)|A secure and optimized binding suitable for cross-machine communication between WCF applications that enables SOAP headers to be used to exchange context.|
+|<xref:System.ServiceModel.WebHttpBinding>|[\<webHttpBinding>](../configure-apps/file-schema/wcf/webhttpbinding.md)|A binding used to configure endpoints for WCF Web services that are exposed through HTTP requests instead of SOAP messages.|
+|<xref:System.ServiceModel.WSHttpContextBinding>|[\<wsHttpContextBinding>](../configure-apps/file-schema/wcf/wshttpcontextbinding.md)|A secure and interoperable binding that is suitable for non-duplex service contracts that enables SOAP headers to be used to exchange context.|
```
Comment (rpetrusha) [inline docs/framework/wcf/system-provided-bindings.md]: same as previous comment

File: docs/framework/wcf/system-provided-bindings.md [hunk 5]
```diff
@@ -1,86 +1,89 @@
...
- [Using Bindings to Configure Services and Clients](../../../docs/framework/wcf/using-bindings-to-configure-services-and-clients.md)  
- [Basic WCF Programming](../../../docs/framework/wcf/basic-wcf-programming.md)
+> <xref:System.ServiceModel.NetHttpBinding> is a binding designed for consuming HTTP or WebSocket services and uses binary encoding by default. <xref:System.ServiceModel.NetHttpBinding> detects whether it is used with a request-reply contract or duplex contract and change its behavior to match - it uses HTTP for request-reply and WebSockets for duplex. This behavior can be overridden using the <xref:System.ServiceModel.NetHttpBinding.WebSocketTransportUsage> binding setting: Allowed - This is the default value and behaves as described above. Never - This prevents WebSockets from being used. Attempting to use a duplex contract with this setting will result in an exception. Always - This forces WebSockets to be used even for request-reply contracts. NetHttpBinding supports reliable sessions in both HTTP mode and WebSocket mode. In WebSocket mode sessions are provided by the transport.
```
Comment (rpetrusha) [inline docs/framework/wcf/system-provided-bindings.md]: - change its behavior --> changes its behavior
- the hyphen between "match" and "it" should be a semicolon.
- the xref is broken; it's \<xref:System.ServiceModel.Channels.WebSocketTransportUsage>. But there's no Allowed value. I think the value is WhenDuplex. See https://docs.microsoft.com/en-us/dotnet/api/system.servicemodel.channels.websockettransportusage?view=netframework-4.7.2.

File: docs/framework/wcf/system-provided-bindings.md [hunk 6]
```diff
@@ -1,86 +1,89 @@
...
+|<xref:System.ServiceModel.WSHttpBinding>|[\<wsHttpBinding>](../configure-apps/file-schema/wcf/wshttpbinding.md)|A secure and interoperable binding that is suitable for non-duplex service contracts.|
+|<xref:System.ServiceModel.WSDualHttpBinding>|[\<wsDualHttpBinding>](../configure-apps/file-schema/wcf/wsdualhttpbinding.md)|A secure and interoperable binding that is suitable for duplex service contracts or communication through SOAP intermediaries.|
+|<xref:System.ServiceModel.WSFederationHttpBinding>|[\<wsFederationHttpBinding>](../configure-apps/file-schema/wcf/wsfederationhttpbinding.md)|A secure and interoperable binding that supports the WS-Federation protocol that enables organizations that are in a federation to efficiently authenticate and authorize users.|
+|<xref:System.ServiceModel.NetHttpBinding>|\<netHttpBinding>|A binding designed for consuming HTTP or WebSocket services that uses binary encoding by default.|
```
Comment (rpetrusha) [inline docs/framework/wcf/system-provided-bindings.md]: This \<netHttpBinding> should be s link. The URL is https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/nethttpbinding.

File: docs/framework/wcf/system-provided-bindings.md [hunk 7]
```diff
@@ -1,86 +1,89 @@
...
+|<xref:System.ServiceModel.NetHttpsBinding>|\<netHttpsBinding>|A secure binding designed for consuming HTTP or WebSocket services that uses binary encoding by default.|
```
Comment (rpetrusha) [inline docs/framework/wcf/system-provided-bindings.md]: This \<netHttpsBinding> should be a link. The URL is https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/wcf/nethttpsbinding.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks good, @mairaw. There's a broken xref, though, and I've left a number of nits and comments,.

**Review by mconnew — APPROVED** [review]

**Comment by mairaw** [comment]
Thanks for your careful review @rpetrusha. I believe I've addressed all your comments.

**Review by rpetrusha — APPROVED** [review]
This looks really good. Thanks for making the additional changes, @mairaw, especially those unrelated to your PR. It's ready to merge.
